applications/openshift/controller/controller_bind_address/rule.yml

```diff
@@ -34,11 +34,15 @@ ocil_clause: |-
     <tt>bind-address</tt> is not configured to a secure IP address
 
 ocil: |-
-    To verify that <tt>bind-address</tt> is configured correctly,
+    To verify that <tt>secure-port</tt> is configured correctly,
```

Comment:
So, this rule is about `bind-address` and not `secure-port`. The change doesn't match the rule and description.

Reply:
Done